presto-tpch/src/main/java/com/facebook/presto/tpch/TpchConnectorFactory.java

```diff
@@ -32,16 +32,25 @@
 public class TpchConnectorFactory
         implements ConnectorFactory
 {
+    public static final boolean DEFAULT_IS_PARTITIONING_ENABLED = false;
```

Comment:
Maybe partitioning could be turned on permanently for all use cases? That would simplify code a bit and I don't see how it could brake anything. Probably this should be discussed with Martin.